src/data/support-title.csv

```diff
@@ -1,65 +1,65 @@
-"[…ただの水滴ですって]",""
-"[一流プランニング]",""
-"[共に同じ道を！]",""
-"[見習い魔女と長い夜]",""
-"[パリピ・ぱーりないと！]",""
-"[準備運動は怠るべからず]",""
-"[むじゃむじゃむじゃき]",""
-"[あなたにささげる]",""
-"[おすすめ本、あります！]",""
-"[世界の真珠、その名は]",""
-"[日本一のステージを]",""
-"[輝く景色の、その先に]",""
-"[夢は掲げるものなのだっ！]",""
-"[不沈艦の進撃]",""
-"[ロード・オブ・ウオッカ]",""
-"[千紫万紅にまぎれぬ一凛]",""
-"[パッションチャンピオーナ！]",""
-"[待望の大謀]",""
-"[天をも切り裂くイナズマ娘！]",""
-"[感謝は指先まで込めて]",""
-"[飛び出せ、キラメケ]",""
-"[B・N・Winner!!]",""
-"[7センチの先へ]",""
-"[Run(my)way]",""
-"[はやい！うまい！はやい！]",""
-"[一粒の安らぎ]",""
-"[これが私のウマドル道☆]",""
-"[まだ小さな蕾でも]",""
-"[うらら～な休日]",""
-"[必殺！Wキャロットパンチ！]",""
-"[ようこそ、トレセン学園へ！]",""
-"[『エース』として]",""
-"[『幸せ』が舞う時]",""
-"[『愛してもらうんだぞ』]",""
-"[夕焼けはあこがれの色]",""
-"[ターボエンジン全開宣言！]",""
-"[バカと笑え]",""
-"[迫る熱に押されて]",""
-"[その背中を越えて]",""
-"[Just keep going.]",""
-"[ふるさと直送エール！]",""
-"[押して忍べど燃ゆるもの]",""
-"[夢はホントに叶うんだ！]",""
-"[今ぞ盛りのさくら花]",""
-"[ふたつのノーブルライト]",""
+"[トレセン学園]","Tracen Academy"
+"[やれやれ、お帰り]","Oh Dear, Welcome Back"
+"[努力は裏切らない！]","Effort Does Not Betray!"
+"[テッペンに立て！]","Stand at the Top!"
+"[副会長の一刺し]","Vice President's Swift Justice"
+"[デジタル充電中+]","Digital Charging in Progress+"
+"[検証、開始]","Verification, Start"
+"[カワイイ＋カワイイは～？]","Cute + Cute is~?"
+"[雨の独奏、私の独創]","Rain's Recital, My Original"
+"[鍛えぬくトモ]","Workout Companion"
+"[鍛えて、応えて！]","Exercise and Excel!"
+"[シチーガール入門♯]","City Girl Training, Begin♯"
+"[生体Aに関する実験的研究]","Experimental Study of Lifeform A"
+"[0500・定刻通り]","0500, Just As Scheduled"
+"[波立つキモチ]","Rippling Feelings"
+"[マーベラス☆大作戦]","Marvelous ☆ Masterplan"
+"[運の行方]","Whereabouts of Luck"
+"[幸せと背中合わせ]","Back-to-Back with Happiness"
+"[目線は気にせず]","Don't Mind the Gazes"
+"[…ただの水滴ですって]","…Just a Drop of Water"
+"[一流プランニング]","First-Class Planning"
+"[共に同じ道を！]","Together on the Same Path!"
+"[見習い魔女と長い夜]","The Apprentice Witch and the Long Night"
+"[パリピ・ぱーりないと！]","Party People, Party Night!"
+"[準備運動は怠るべからず]","Thou Shan't Neglect Warming Up"
+"[むじゃむじゃむじゃき]","Disheveled Innocence"
+"[あなたにささげる]","Devoted to You"
+"[おすすめ本、あります！]","Book Recommendations Available!"
+"[世界の真珠、その名は]","Pearl of the World"
+"[日本一のステージを]","Japan's Number 1 Stage"
+"[輝く景色の、その先に]","Beyond the Brilliant Scenery"
+"[夢は掲げるものなのだっ！]","Let Your Dreams Be Known!"
+"[不沈艦の進撃]","Advance of the Unsinkable Battleship"
+"[ロード・オブ・ウオッカ]","Road of Vodka"
+"[千紫万紅にまぎれぬ一凛]","A Myriad of Colors, A Noble Bloom"
+"[パッションチャンピオーナ！]","Passion Championa!"
+"[待望の大謀]","Long-awaited Masterplan"
+"[天をも切り裂くイナズマ娘！]","Heaven-piercing Lightning Girl!"
+"[感謝は指先まで込めて]","Gratitude up to one's Fingertips"
+"[飛び出せ、キラメケ]","Jump Out and Shine"
+"[B・N・Winner!!]","B・N・Winner!!"
+"[7センチの先へ]","Beyond the 7 cm"
+"[Run(my)way]","Run (my) Way"
+"[はやい！うまい！はやい！]","Fast! Skillful! Fast!"
+"[一粒の安らぎ]","A Grain of Peace"
+"[これが私のウマドル道☆]","This is my 'Umadol' Way ☆"
+"[まだ小さな蕾でも]","Even as a Tiny Bud"
+"[うらら～な休日]","Urara~n Holiday"
+"[必殺！Wキャロットパンチ！]","Special Move! W Carrot Punch!"
+"[ようこそ、トレセン学園へ！]","Welcome to Tracen Academy!"
+"[『エース』として]","As the Ace"
+"[『幸せ』が舞う時]","When Happiness Dances"
+"[『愛してもらうんだぞ』]","'You'll Be Dearly Beloved'"
```

Comment:
Are the apostrophes necessary? It's not a quote or anything afaik, it's just gonna look weird in-game

Reply:
In the last one, it is absolutely necessary. It is definitely a quote if you read the card's episode text.

Reply:
Yeah you're right, keep them then, I'd just personally change them to double quotation marks to keep the writing style consistent though

Reply:
Sadly using single quotes instead of double quotes is necessary because escaping quotes via backslash doesn't work.

Reply:
Some more info on this: https://github.com/ClickHouse/ClickHouse/issues/10624#issuecomment-622971581

Using two "s also didn't work for escaping, tested in a previous PR.

Reply:
We're not using [standardised csv parsing logic at all](https://github.com/FabulousCupcake/umamusume-db-translate/blob/master/src/csvtojson.jq) so it might work

Reply:
<img width="306" alt="Screen Shot 2021-05-20 at 14 55 44" src="https://user-images.githubusercontent.com/25855364/118982184-77629f00-b97b-11eb-92f9-c428a47c1ba3.png">

Reply:
It works

Reply:
Or did you mean that it breaks github's special csv render?

Reply:
<img width="697" alt="Screen Shot 2021-05-20 at 14 58 53" src="https://user-images.githubusercontent.com/25855364/118982646-ec35d900-b97b-11eb-907b-8f7b614ffc29.png">

Reply:
sigh, github pls

Reply:
Nah, I was actually mistaken. What did indeed not work when tested ingame was using two "s, e.g. ""Test"" where it should show up ingame as "Test". Can be seen in https://github.com/FabulousCupcake/umamusume-db-translate/pull/47#issuecomment-833317735.

So you're saying Backslash escaping works, if I'm getting it right?

Reply:
It does not. Github complains about it, which is not acceptable; the reason why it's csv in the first place is because github can show it nicely

Reply:
If we do want to use it, we can use something like `&quot;` and modify `csvtojson.jq` file to search and replace them into `"` while it's made into json, but that can (and should be) a separate pr.

Reply:
I'll keep it as single quotes for now then.